### ISSUE TITLE
wr.s3.read_parquet fails at _apply_timezone on version >= 1.9.2

### ISSUE THREAD

**Issue by Thiago-Dantas** [awslabs/aws-data-wrangler#417]
**Describe the bug**
When reading a list of parquet files I get the traceback

```
....
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 291, in _read_parquet_chunked
    path_root=path_root,
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 236, in _arrowtable2df
    df = _apply_timezone(df=df, metadata=metadata)
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 194, in _apply_timezone
    if c["field_name"] in df.columns and c["pandas_type"] == "datetimetz":
KeyError: 'field_name'
```
The same happens when not chunking files
```
....
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 384, in _read_parquet
    path_root=path_root,
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 241, in _arrowtable2df
    df = _apply_timezone(df=df, metadata=metadata)
  File "/usr/local/lib/python3.6/dist-packages/awswrangler/s3/_read_parquet.py", line 199, in _apply_timezone
    if c["field_name"] in df.columns and c["pandas_type"] == "datetimetz":
KeyError: 'field_name'
```

When debugging and inspecting `c`, it has no `field_name` key, but has a `name` key
```
def _apply_timezone(df: pd.DataFrame, metadata: Dict[str, Any]) -> pd.DataFrame:
    for c in metadata["columns"]:
        if c["field_name"] in df.columns and c["pandas_type"] == "datetimetz":
```
![image](https://user-images.githubusercontent.com/6326626/95245540-a744a400-07e9-11eb-8933-1d3d1be8be98.png)

My actual reading is as simple as it gets
```
def _read_parquet(file_path: list[str]):
    dfs = wr.s3.read_parquet(
        path=file_path
    )
    return dfs
```

For now we reverted back to 1.9.1 to get it working again since our parquet files don't really have datetimetz fields

**Comment by igorborgest** [comment]
Thank you @Thiago-Dantas! Really useful informations!

I've just fixed this issue in the commit above.

Do you mind to test it from our development branch and check if it helps in your use case?
The ideia is to publish this enhancement in the version `1.9.6` this weekend.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by Thiago-Dantas** [comment]
Thank you for the super fast response

Could not reproduce my error on the dev branch, thanks!

**Comment by igorborgest** [comment]
Released in [1.9.6](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.9.6)!
